projects/novo-elements/src/elements/data-table/interfaces.ts

```diff
@@ -20,7 +20,7 @@ export interface IDataTablePreferences {
 export interface DataTableWhere {
   query: string;
   criteria?: AdaptiveCriteria;
-  keywords?: string[];
+  keywords?: SearchKeywords;
```

Comment:
i don't think this update is going to be backwards compatible.

Reply:
this type was added recently enough that we're confident no one else is using it, so we're going to move forward with this update anyway.